mmv1/third_party/terraform/services/identityplatform/resource_identity_platform_config_test.go

```diff
@@ -82,6 +82,13 @@ resource "google_identity_platform_config" "basic" {
         }
     }
   }
+  sms_region_config {
```

Comment:
Can you please add `sms_region_config` with `allow_by_default` or `allowlist_only` to the example? https://github.com/GoogleCloudPlatform/magic-modules/blob/4b727fa6623e03eef4774ddfaae84b9b0cc74202/mmv1/templates/terraform/examples/identity_platform_config_basic.tf.erb

Reply:
Done!